docs/reference/sdk/classes/core.PackDefinitionBuilder.md

```diff
@@ -26,17 +26,13 @@ rather than constructing a builder directly.
 | :------ | :------ |
 | `definition?` | `Partial`<[`PackVersionDefinition`](../interfaces/core.PackVersionDefinition.md)\> |
 
-#### Defined in
```

Comment:
Is "Defined in" going away just the way typedoc behaves now, or is there some setting that controls whether this appears?

Seems like a person could still find the source from the links below each member, but I kinda think these links are handy too.